daemon/daemon_windows.go

```diff
@@ -601,3 +606,45 @@ func (daemon *Daemon) verifyVolumesInfo(container *container.Container) error {
 func (daemon *Daemon) setupSeccompProfile() error {
 	return nil
 }
+
+func waitOOBEComplete() error {
```

Comment:
I think pkg/system is generally for platform agnostic versions of things. Not to say I would be against moving it makes sense there though.